docs/deploy/howto-deploy-aks.md

```diff
@@ -514,34 +514,80 @@ Command line argument key-value pairs can be specified with:
 
 ### AKS
 
-All of microservices of Industrial IoT solution are deployed to an AKS Kubernetes cluster. The deployment
-creates `industrial-iot` namespace where all microservices are running. To provide connection details for
-all Azure resources created by `Microsoft.Azure.IIoT.Deployment`, we created `industrial-iot-env` secret,
-which is then consumed by deployments.
+All of microservices of Industrial IoT solution are deployed to an AKS Kubernetes cluster.
+
+#### Resource Definitions
+
+The deployment creates `industrial-iot` namespace where all microservices are running. To provide connection
+details for all Azure resources created by `Microsoft.Azure.IIoT.Deployment`, we created
+`industrial-iot-env` secret, which is then consumed by deployments.
 
 To see YAML files of all Kubernetes resources that are created by the application check
-[deploy/src/Microsoft.Azure.IIoT.Deployment/Resources/aks/](../deploy/src/Microsoft.Azure.IIoT.Deployment/Resources/aks/)
+[deploy/src/Microsoft.Azure.IIoT.Deployment/Resources/aks/](../../deploy/src/Microsoft.Azure.IIoT.Deployment/Resources/aks/)
 directory.
 
-To see state of microservices you can check Kubernetes Dashboard. Follow this tutorial on how to access it:
+#### Kubernetes Dashboard
 
-* [Access the Kubernetes web dashboard in Azure Kubernetes Service (AKS)](https://docs.microsoft.com/azure/aks/kubernetes-dashboard)
+To see state of microservices you can check Kubernetes dashboard.
+
+You can follow this tutorial to do that: [Access the Kubernetes web dashboard in Azure Kubernetes Service (AKS)](https://docs.microsoft.com/azure/aks/kubernetes-dashboard).
+
+Alternatively, you can run the following commands:
+
+1. Make sure you have `kubectl` installed. If it is not already installed, run the following command to
+    download and install `kubectl`:
+
+    ```bash
+    az aks install-cli
+    ```
+
+    More details about the command and its optional parameters can be found [here](https://docs.microsoft.com/cli/azure/aks?view=azure-cli-latest#az-aks-install-cli).
+
+2. Get access credentials for a managed Kubernetes cluster. You would need name of the resource group
+    where AKS resource is and the name of AKS resource itself:
```

Comment:
You would need the name of the resource group containing AKS resource and the name of the AKS resource:

Reply:
Done.